spec/services/no_bp_measure_service_spec.rb

```diff
@@ -5,51 +5,55 @@
   let(:user) { create(:admin, :supervisor, organization: organization) }
   let(:facility_group_1) { FactoryBot.create(:facility_group, name: "facility_group_1", organization: organization) }
 
-  let(:june_1_2018) { Time.parse("June 1, 2018 00:00:00+00:00") }
-  let(:june_1_2020) { Time.parse("June 1, 2020 00:00:00+00:00") }
-  let(:june_30_2020) { Time.parse("June 30, 2020 00:00:00+00:00") }
-  let(:july_2020) { Time.parse("July 15, 2020 00:00:00+00:00") }
-  let(:jan_2019) { Time.parse("January 1st, 2019 00:00:00+00:00") }
-  let(:jan_2020) { Time.parse("January 1st, 2020 00:00:00+00:00") }
-  let(:july_2018) { Time.parse("July 1st, 2018 00:00:00+00:00") }
-  let(:july_2020) { Time.parse("July 1st, 2020 00:00:00+00:00") }
-
```

Comment:
@timcheadle any reason you got rid of these? i believe Hari added the timezone info to try to catch TZ boundary condition issues.

Reply:
They weren't being used anywhere?

Reply:
Oh.  thats a good reason =)  I thought this was a different spec, my mistake.